AutomatedTesting/Gem/PythonTests/AWS/Windows/client_auth/aws_client_auth_automation_test.py

```diff
@@ -86,16 +96,29 @@ def test_password_signin_credentials(self,
         """
         Test to verify AWS Cognito IDP Password sign in and Cognito Identity pool authenticated authorization.
 
-        Setup: Deploys cdk and updates resource mapping file.
+        Setup: Updates resource mapping file using existing CloudFormation stacks.
         Tests: Sign up new test user, admin confirm the user, sign in and get aws credentials.
         Verification: Log monitor looks for success credentials log.
         """
         asset_processor.start()
         asset_processor.wait_for_idle()
 
-        file_to_monitor = os.path.join(launcher.workspace.paths.project_log(), GAME_LOG_NAME)
+        file_to_monitor = os.path.join(launcher.workspace.paths.project_log(), constants.GAME_LOG_NAME)
         log_monitor = ly_test_tools.log.log_monitor.LogMonitor(launcher=launcher, log_file_path=file_to_monitor)
 
+        cognito_idp = aws_utils.client('cognito-idp')
+        user_pool_id = resource_mappings.get_resource_name_id(f'{AWS_CLIENT_AUTH_FEATURE_NAME}.CognitoUserPoolId')
+        print(f'UserPoolId:{user_pool_id}')
```

Comment:
nit: If this is useful should it go through the logger?

Reply:
Will do.